### PR TITLE
chore(infra-apps): bump velero to 2.32.4

### LINKED ISSUE #865: Version 2.32.3 of velero is available!
Hi!<br/></br>There is a new version available for velero!<br/><br/>current targetRevision set: `2.31.*`<br/>new version available: `2.32.3`<br/><br/>Please consider updating the `targetRevision` with a PR in this file [values.yaml](https://github.com/adfinis/helm-charts/tree/master/charts/infra-apps/values.yaml)<br/><br/>Thanks.

### PR DESCRIPTION
**Pull request by vmaillot**
# Description

Bump Infra-apps to `0.136.0` to use Velero `1.9.4` with chart `2.32.4`.

Useful changes:
* https://github.com/vmware-tanzu/helm-charts/issues/365 
* https://github.com/vmware-tanzu/helm-charts/pull/424

# Issues

Fixes #865.

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated the changelog according to the `artifacthub.io/changes` annotation in the Chart.yaml. See following [example](https://github.com/adfinis/helm-charts/pull/858/files).
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

**Review by Nerkho — APPROVED** [review]
LGTM

File: charts/infra-apps/Chart.yaml [hunk 1]
```diff
@@ -21,3 +21,8 @@ annotations:
       links:
```
Comment (isantospardo) [inline charts/infra-apps/Chart.yaml:21]: I guess you have to delete the old changes to only have the last entry in here with this specific change/update

Comment (vmaillot) [reply charts/infra-apps/Chart.yaml:21]: oh really?

Comment (vmaillot) [reply charts/infra-apps/Chart.yaml:21]: well as it is versionned it makes total sense, lemme update this

**Review by Nerkho — APPROVED** [review]
LGTM

**Review by isantospardo — APPROVED** [review]

**Pull request merged**